pkg/volume/nfs/nfs.go

```diff
@@ -138,10 +143,11 @@ func (plugin *nfsPlugin) NewUnmounter(volName string, podUID types.UID) (volume.
 
 func (plugin *nfsPlugin) newUnmounterInternal(volName string, podUID types.UID, mounter mount.Interface) (volume.Unmounter, error) {
 	return &nfsUnmounter{&nfs{
-		volName: volName,
-		mounter: mounter,
-		pod:     &v1.Pod{ObjectMeta: metav1.ObjectMeta{UID: podUID}},
-		plugin:  plugin,
+		volName:         volName,
+		mounter:         mounter,
+		pod:             &v1.Pod{ObjectMeta: metav1.ObjectMeta{UID: podUID}},
+		plugin:          plugin,
+		MetricsProvider: volume.NewMetricsStatFS(getPath(podUID, volName, plugin.host)),
```

Comment:
`NewMetricsStatFS` calls `FsInfo`. According to NFS spec, FSINFO [1] call doesn't return capacity, inode, free inodes, etc. Additionally, NFS FSSTAT call [2] doesn't return these info either. 

For reference, there was an early attempt to get NFS metrics through `du` [3], but there was a performance concern


1. https://www.freesoft.org/CIE/RFC/1813/39.htm
2. https://www.freesoft.org/CIE/RFC/1813/38.htm
3. https://github.com/kubernetes/kubernetes/pull/33468/files

Reply:
Can the performance problem mitigated by collection this metric less often (or caching it..)?

Reply:
`du` on NFS can take really long time (hours) on big volumes...

`statfs()` returns something useful, at least with a Linux NFS server.

Reply:
There is a thought that I can use a switch like 'EnableMetrics' to conditionally pull statfs data but then I need to add a bool flag someplace like nfs VolumeSource. If that is acceptable then I can work on it.